msg/Detection2DWithCompressionArray.msg

```diff
@@ -0,0 +1,7 @@
+# A list of 2D detections, for a multi-object 2D detector.
```

Comment:
Is it worth explicitly mentioning the compressed images in the header?

e.g. `A list of 2D detections with compressed images, for a multi-object 2D detector.`